registry/xml/al.xml

```diff
@@ -0,0 +1,5469 @@
+            <param><ptype>ALCuint</ptype> <name>frequency</name></param>
+            <param group="Format"><ptype>ALCenum</ptype> <name>format</name></param>
+            <param><ptype>ALCsizei</ptype> <name>buffersize</name></param>
+        </command>
+        <command export="alc" comment="Closes the given capture device.">
+            <proto><ptype>ALCboolean</ptype> <name>alcCaptureCloseDevice</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+        </command>
+        <command export="alc" comment="Starts capturing samples into the device buffer.">
+            <proto>void <name>alcCaptureStart</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+        </command>
+        <command export="alc" comment="Stops capturing samples. Samples in the device buffer remain available.">
+            <proto>void <name>alcCaptureStop</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+        </command>
+        <command export="alc" comment="Reads samples from the device buffer.">
+            <proto>void <name>alcCaptureSamples</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+            <param><ptype>ALCvoid</ptype> *<name>buffer</name></param>
+            <param><ptype>ALCsizei</ptype> <name>samples</name></param>
+        </command>
+        <command export="alc">
+            <proto><ptype>ALCdevice</ptype>* <name>alcLoopbackOpenDeviceSOFT</name></proto>
+            <param>const <ptype>ALCchar</ptype> *<name>deviceName</name></param>
+        </command>
+        <command export="alc">
+            <proto><ptype>ALCboolean</ptype> <name>alcIsRenderFormatSupportedSOFT</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+            <param><ptype>ALCsizei</ptype> <name>freq</name></param>
+            <param group="RenderFormatChannelSOFT"><ptype>ALCenum</ptype> <name>channels</name></param>
+            <param group="RenderFormatTypeSOFT"><ptype>ALCenum</ptype> <name>type</name></param>
+        </command>
+        <command export="alc">
+            <proto>void <name>alcRenderSamplesSOFT</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+            <param><ptype>ALCvoid</ptype> *<name>buffer</name></param>
+            <param><ptype>ALCsizei</ptype> <name>samples</name></param>
+        </command>
+        <command export="alc">
+            <proto>void <name>alcDevicePauseSOFT</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+        </command>
+        <command export="alc">
+            <proto>void <name>alcDeviceResumeSOFT</name></proto>
+            <param><ptype>ALCdevice</ptype> *<name>device</name></param>
+        </command>
+        <command export="alc">
+            <proto>const <ptype>ALchar</ptype>* <name>alcGetStringiSOFT</name></proto>
```

Comment:
Should be `ALCchar` instead of `ALchar`.

Reply:
Fixed, thanks.